tests/lib/boot.sh

```diff
@@ -47,9 +47,7 @@ get_boot_path() {
     elif [ -f /boot/grub/grubenv ]; then
         echo "/boot/grub/"
     else
-        echo "Cannot determine boot path"
-        ls -alR /boot
-        exit 1
+        echo "/boot/"
```

Comment:
I think the UC20 uboot case should be handled like so:
```
get_boot_path() {
    if [ -f /boot/uboot/uboot.env -o -f /boot/uboot/uboot.sel ]; then
        # uc16/uc18 have /boot/uboot/uboot.env
        # uc20 has /boot/uboot/uboot.sel
        echo "/boot/uboot/"
    elif [ -f /boot/grub/grubenv ]; then
        echo "/boot/grub/"
    else
        echo "Cannot determine boot path"
        ls -alR /boot
        exit 1
    fi
}
```

Reply:
@bboozzoo Thanks for the suggestion, the fix is already pushed.